requirements/fabric/base.txt

```diff
@@ -2,7 +2,7 @@
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
 
 numpy >=1.17.2, <1.27.0
-torch >=1.11.0, <2.1.0
+torch >=1.12.0, <2.1.0
```

Comment:
Don't we also have in some req. `torchvision` which needs to be bumped also

Reply:
Thanks for the reminder. I bumped it to >= 1.13 which is the version supported for PyTorch 1.12 given the version table here in the readme: https://github.com/pytorch/vision